bin/match_zcat_photo

```diff
@@ -0,0 +1,696 @@
+#!/usr/bin/env python
+
+"""Match data release redshift catalogs against the original photometric
+(target) catalogs.
+
+Sequence of commands for a given release (here, fuji)
+1. Gather targeting photometry from the individual redshift catalogs.
+
+   time match_zcat_photo --reduxdir /global/cfs/cdirs/desi/spectro/redux/fuji -o /global/cscratch1/sd/ioannis/photocatalog/fuji --outprefix fuji --mp 32 --targetphot
+
+   Output files:
+    targetphot-fuji.fits
+    targetphot-zcat-fuji.fits
+    targetphot-missing-fuji.fits
+
+2. Gather Tractor photometry.
+
+   time match_zcat_photo --reduxdir /global/cfs/cdirs/desi/spectro/redux/fuji -o /global/cscratch1/sd/ioannis/photocatalog/fuji --outprefix fuji --mp 32 --tractorphot
+
+   Output files:
+     tractorphot-nside4-hp???-fuji.fits
+
+3. Validate Tractor photometry
+
+   time match_zcat_photo --reduxdir /global/cfs/cdirs/desi/spectro/redux/fuji -o /global/cscratch1/sd/ioannis/photocatalog/fuji --outprefix fuji --validate-tractorphot
+
+   Output files:
+     tractorphot-validate-fuji.fits
+
+4. Gather targeting and Tractor photometry for *potential* targets (going back to the original fiberassign files).
+
+   time match_zcat_photo --reduxdir /global/cfs/cdirs/desi/spectro/redux/fuji -o /global/cscratch1/sd/ioannis/photocatalog/fuji --outprefix fuji --mp 32 --targetphot --potential-targets
+   time match_zcat_photo --reduxdir /global/cfs/cdirs/desi/spectro/redux/fuji -o /global/cscratch1/sd/ioannis/photocatalog/fuji --outprefix fuji --mp 32 --tractorphot --potential-targets
+
+   Output files:
+     targetphot-potential-targets-fuji.fits
+     targetphot-potential-targets-zcat-fuji.fits
+     targetphot-potential-targets-missing-fuji.fits
+     tractorphot-potential-targets-nside4-hp???-fuji.fits
+
+"""
```

Comment:
Are all of the functions in this script very specific to this script itself, or should any of them be pulled out into a module for 3rd party use like you did for the `desispec.io.photo` functions?

Reply:
Everything in this top-level script is pretty specific to this task.